e2e/tests/performance/imagery.perf.spec.js

```diff
@@ -0,0 +1,173 @@
+        await page.evaluate(() => (window.performance.measure("machine-time-difference", "viewlarge.start", "viewLarge.start.test")));
+
+        //Get All Performance Marks
+        const getAllMarksJson = await page.evaluate(() => JSON.stringify(window.performance.getEntriesByType("mark")));
+        const getAllMarks = JSON.parse(getAllMarksJson);
+        console.log('window.performance.getEntriesByType("mark")', getAllMarks);
+
+        //Get All Performance Measures
+        const getAllMeasuresJson = await page.evaluate(() => JSON.stringify(window.performance.getEntriesByType("measure")));
+        const getAllMeasures = JSON.parse(getAllMeasuresJson);
+        console.log('window.performance.getEntriesByType("measure")', getAllMeasures);
+
+    });
+    /* The following test will navigate to a previously created Performance Display Layout and measure the
+    /  following metrics:
+    /  - ElementResourceTiming
+    /  - Interaction Timing
+    */
+    test('Embedded View Large for Imagery is performant in Fixed Time', async ({ page, browser }) => {
+        const client = await page.context().newCDPSession(page);
+        // Tell the DevTools session to record performance metrics
+        // https://chromedevtools.github.io/devtools-protocol/tot/Performance/#method-getMetrics
+        await client.send('Performance.enable');
+        // Go to baseURL
+        await page.goto('/');
+
+        // To to Search Available after Launch
+        await page.locator('input[type="search"]').click();
+        await page.evaluate(() => (window.performance.mark("search-available")));
+        // Fill Search input
+        await page.locator('input[type="search"]').fill('Performance Display Layout');
+        await page.evaluate(() => (window.performance.mark("search-entered")));
+        //Search Result Appears and is clicked
+        await Promise.all([
+            page.waitForNavigation(),
+            page.locator('a:has-text("Performance Display Layout")').first().click(),
+            page.evaluate(() => (window.performance.mark("click-search-result")))
+        ]);
+
+        //Time to Example Imagery Frame loads within Display Layout
+        await page.waitForSelector('.c-imagery__main-image__bg', { state: 'visible'});
+        //Time to Example Imagery object loads
+        await page.waitForSelector('.c-imagery__main-image__background-image', { state: 'visible'});
+
+        //Get background-image url from background-image css prop
+        const backgroundImage = await page.locator('.c-imagery__main-image__background-image');
+        let backgroundImageUrl = await backgroundImage.evaluate((el) => {
+            return window.getComputedStyle(el).getPropertyValue('background-image').match(/url\(([^)]+)\)/)[1];
+        });
+        backgroundImageUrl = backgroundImageUrl.slice(1, -1); //forgive me, padre
+        console.log('backgroundImageurl ' + backgroundImageUrl);
```

Comment:
@ozyx on e2e tests, I'll leave console logs in place on points in tests which themselves are not asserting on anything. Since these will be running on a wide array of environments, it'll be important to get diagnostic information on assumptions that are made in the open vs closed tests.